lib/VCMIDirs.cpp

```diff
@@ -458,6 +438,14 @@ bfs::path VCMIDirsOSX::userDataPath() const
 bfs::path VCMIDirsOSX::userCachePath() const { return userDataPath(); }
 bfs::path VCMIDirsOSX::userConfigPath() const { return userDataPath() / "config"; }
 
+bfs::path VCMIDirsOSX::userLogsPath() const
+{
+	// TODO: use proper objc code from Foundation framework
+	if(const auto homeDir = std::getenv("HOME"))
+		return bfs::path{homeDir} / "Library" / "Logs" / "vcmi";
```

Comment:
It make sense for linux/ios/android, but what the reason to change path for mac? Game data anyway is in application support and game creates artifacts such as downloads, settings. I suggest do not change log path for mac

Reply:
There're standard OS-specific things that should be respected. Logs directory exists for a reason.